core-java-modules/core-java-security-2/src/main/java/com/baeldung/aes/AESUtil.java

```diff
@@ -0,0 +1,183 @@
+package com.baeldung.crypto;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.crypto.*;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.PBEKeySpec;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.*;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.KeySpec;
+import java.util.Base64;
+
+public class AESUtil {
+    static Logger logger = LoggerFactory.getLogger(AESUtil.class);
```

Comment:
I guess we can make the `logger` private and final as well and have it in capitals?